src/panel/direction/DirectionPanel.jsx

```diff
@@ -372,6 +400,18 @@ export default class DirectionPanel extends React.Component {
           renderHeader={form}
         >
           <div id="direction-autocomplete_suggestions" />
+          <ShareMenu url={window.location.toString()}>
+            {openMenu => <Button
+              className="u-ml-auto u-flex-shrink-0"
```

Comment:
u-flex-shrink-0 is still needed to avoid this behavior:
![Peek 2020-10-29 16-22](https://user-images.githubusercontent.com/2981774/97594415-f2299400-1a02-11eb-9627-225c500d208e.gif)